google/cloud/storage/internal/read_object_range_request.cc

```diff
@@ -0,0 +1,76 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "google/cloud/storage/internal/read_object_range_request.h"
+#include <sstream>
+
+namespace storage {
+inline namespace STORAGE_CLIENT_NS {
+namespace internal {
+ReadObjectRangeResponse ReadObjectRangeResponse::FromHttpResponse(
+    HttpResponse&& response) {
+  auto loc = response.headers.find(std::string("content-range"));
+  if (response.headers.end() == loc) {
+    google::cloud::internal::RaiseInvalidArgument(
+        "invalid http response for ReadObjectRange");
+  }
+
+  std::string const& header = loc->second;
```

Comment:
Readability nit (from dealing with this in lots of other libraries): Can you give this a more descriptive name than "header"?  Also, "header" generally refers to the entire header field ("field-name: field-value"), so it's helpful to be specific about which portion you're grabbing (even though it's fairly obvious from `loc->second` that this is referring to the value).

Could you call this something like `content_range_value`?

Reply:
Fixed.